src/fibonacci.js

```diff
@@ -1,4 +1,5 @@
 'use strict'
+
 export default function fibonacci (number) {
 var fibArray = [0,1]
```

Comment:
`let` instead of `var` if you're going to initialize the variable to something else down the line.